apps/web/app/(app)/environments/[environmentId]/surveys/components/PreviewSurvey.tsx

```diff
@@ -361,7 +375,7 @@ export default function PreviewSurvey({
       </div>
     </div>
   );
-}
+};
 
 function ResetProgressButton({ resetQuestionProgress }) {
   return (
```

Comment:
Can we either move this component outside? or please change it to arrow function.

Reply:
It was being used at multiple places, so created a component for it in UI package 😊